pkg/adsc/adsc_test.go

```diff
@@ -185,7 +184,7 @@ func TestADSC_Run(t *testing.T) {
 				t.Errorf("ADSC: failed running %v", err)
 				return
 			}
-			assert.Eventually(t, func() bool {
+			assertEventually(t, func() bool {
```

Comment:
can we use EventuallyEqual in   `pkg/test/util/assert`

Reply:
Cool, changed. Thanks.